### PR TITLE
DDF-04746 eslint: turn on no-absolute-path-references (previously named no-absolute-urls)

### PR DESCRIPTION
**Pull request by rymach**
this needs to be ON so the app can be deployed in a reverse proxy 
@brendan-hofmann 
https://serverfault.com/a/561897

upgraded to latest ace

### CODE REVIEW

**Comment by rymach** [comment]
build now

File: ui/.eslintrc.js [hunk 1]
```diff
@@ -1,8 +1,6 @@
 module.exports = {
     extends: ["@connexta/eslint-config-connexta", "eslint:recommended", "plugin:react/recommended"],
     rules: {
-        "@connexta/connexta/no-absolute-urls": "off",
```
Comment (Bdthomson) [inline ui/.eslintrc.js]: Is this on somewhere else by default?

Comment (rymach) [reply ui/.eslintrc.js]: `"@connexta/eslint-config-connexta",`

**Comment by rymach** [comment]
don't merge this yet

**Comment by rymach** [comment]
build now

**Comment by rymach** [comment]
build now

**Comment by rymach** [comment]
build now

**Comment by rymach** [comment]
build now

**Comment by rymach** [comment]
this is ready for merge

**Comment by bdeining** [comment]
build now

**Pull request merged**